index.bs

```diff
@@ -328,16 +328,17 @@ When this method is invoked, the user agent MUST run the following steps:
 
   1. Let |promise| be [=a new Promise=].
   1. Let |immersive| be <code>true</code> if |mode| is an [=immersive session=] mode, and <code>false</code> otherwise.
+  1. Let |global object| be the [=relevant Global object=] for the {{Navigator}} object which the {{XRSystem}} on which this method was invoked is a member of. 
```

Comment:
The relevant global is probably the right global here. I think the language can be simplified to "the [=relevant Global object=] for the {{XRSystem}} " (all platform objects have a relevant global, you don't need to say the navigator).